infrastructure/aat.tfvars

```diff
@@ -2,4 +2,5 @@ root_logging_level = "DEBUG"
 log_level_spring_web = "DEBUG"
 log_level_sscs = "DEBUG"
 management_security_enabled = "false"
-idam_oauth2_redirect_url = "https://sscs-case-loader-aat.service.core-compute-aat.internal"
```

Comment:
@bartlabno , we have you put this variable? It's needed to connect to CCD via IdAM.

Reply:
There is no need for this variable as it is replaced by: https://github.com/hmcts/sscs-case-loader/pull/53/files#diff-1031eeab417cd8afe77e7e005717d38dR71